src/utils/lod-validation.ts

```diff
@@ -0,0 +1,46 @@
+import type { Tile3D } from "@loaders.gl/tiles";
```

Comment:
I suggest moving this file in `tile-validation` folder.

Reply:
Done